src/assets/RawAsset.js

```diff
@@ -6,8 +6,11 @@ class RawAsset extends Asset {
   load() {}
 
   generate() {
+    const pathToAsset = JSON.stringify(
+      path.join(this.options.publicURL, this.generateBundleName())
```

Comment:
will this cause a `\` to be used on windows instead of `/`?

Reply:
you probably want `url.resolve` https://nodejs.org/api/url.html#url_url_resolve_from_to

Reply:
Thanks  for the heads up. On it.

Reply:
Done. I had misunderstood how modules are loaded. Thanks for